.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Description
+
+What changed? Why? (the code + comments should speak for itself on the "how")
+
+Fun screenshots ir a cool video or something are super helpful as well. If this touches platform-specific behavior, this is where test evidence should be collected.
+
+Any issues this pull request closes or pull requests this supersedes should be linked with `Closes #issuenumber`.
+
+## Meta
+
+Merge checklist:
+- [ ] Pull Request title is [short, imperative summary](https://cbea.ms/git-commit/) of proposed changes
+- [ ] The description documents the _what_ and _why_
+- [ ] If this PR touches photon-serde, all messages have been regenerated and hashes have not changed unexpectedly
+- [ ] If this PR touches configuration, this is backwards compatible with settings back to v2024.3.1
```

Comment:
should it be for that year? 2025.X.X or whatever unless this is just intentional to make it always compatible back to then

Reply:
The current update plan is to always be able to support configs from last years last stable build.

Reply:
Not related to this PR, but is there a test for loading the last stable config?

Reply:
yes added in https://github.com/PhotonVision/photonvision/pull/1487

Reply:
Yep, the intent is that this template would be updated when we need to bump that.